pymc3/distributions/continuous.py

```diff
@@ -1081,14 +1154,18 @@ class ExGaussian(Continuous):
         Tutorials in Quantitative Methods for Psychology,
         Vol. 4, No. 1, pp 35-45.
     """
-    def __init__(self, mu, sigma, nu, *args, **kwargs):
-        super(ExGaussian, self).__init__(*args, **kwargs)
-        self.mu = mu
-        self.sigma = sigma
-        self.nu = nu
+
+    def __init__(self, mu, sigma, nu, ndim=None, size=None, dtype=None, *args, **kwargs):
```

Comment:
To fix the current issues, which seem to revolve around producing a final/total shape, it might actually be ideal to already have the default value, since we can just use its shape.  Again, that would require implementations to provide the default value before calling the base `Distribution` constructor.
@twiecki, if that's a reasonable requirement, I'll just implement that now.